### PR TITLE
fix #81: restore close button to toolbar

### LINKED ISSUE #81: Add generic "close" button to toolbar
Just spotted this, and thought adding a "close" button may be nice, if user's just want to dismiss the Block header completely without doing anything.

firefox_test_pilot_-_experiments

### PR DESCRIPTION
**Pull request by groovecoder**
blok-close-btn

### CODE REVIEW

**Comment by groovecoder** [comment]
@pdehaan - check this out?

File: toolbar.html [hunk 1]
```diff
@@ -12,29 +12,32 @@
...
-        <div id="feedback-page-works" class="feedback hide">
+        </span>
+        <span id="feedback-page-works" class="feedback hide">
           You said this page works well.
           <a class="feedback-btn tiny button" data-feedback="page-problem">Report a problem</a>
-        </div>
+        </span>
+        <span>
+          <a id="close-btn" class="tiny button close-btn">X</a>
+        </span>
```
Comment (pdehaan) [inline toolbar.html]: Functionality seems good to me. I click a button and a thing goes away. Not sure if the UI/UX person wants to weigh in on the hover states, etc.

I'm also not a Foundation Person, so I'm not :100: on our close button implementation versus something like their canned "Close Button" http://foundation.zurb.com/sites/docs/close-button.html

**Comment by pdehaan** [comment]
### Totally looks like:

firefox_test_pilot
**Figure 1:** Regular.

---

firefox_test_pilot
**Figure 2:** Hovered.

**Comment by groovecoder** [comment]
Yeah I wanted to get a close button added. If we need to update its styles to match product UI I can dig into that - either before I merge here or as a follow-up issue.

**Comment by pdehaan** [comment]
@groovecoder Sounds great. Let's land this and we can do a follow-up if UX disapproves.

Also, can you make sure @rbillings is added to this repo. I tried reassigning this to her and failed, and I think I lack the admin levels to add her as a collaborator/admin, or however GitHub works.

**Comment by groovecoder** [comment]
I added @rbillings to the repo and assigned.

**Comment by rbillings** [comment]
Verified that close button displays, and removes toolbar when clicked. Toolbar reappears with a refresh.
* The first two Dev Edition browsers I opened were slow to disable/enable on all tabs. It also required multiple clicks of "disable" for each tab. This no longer repros.
* The toolbar has a different display pattern, this does repro: 
** No toolbar for cbs.com & nbc.com
** Toolbar displays on https://www.mozilla.org/en-US/about/
** Other sites such as youtube, cnn, huffpo, etc. display as expected

**Comment by groovecoder** [comment]
@rbillings - good catches. I had a flaky if condition for checking when to create the toolbar. Think I've got it fixed now.

**Comment by rbillings** [comment]
Verified close button & correct toolbar displays.

**Pull request merged**